examples/controllers/Docker/init.sql

```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE IF NOT EXISTS controllers;
```

Comment:
could bi xiana-dev, couldn't?

Reply:
> could be xiana-dev the db couldn't?

I don't want to touch the existing code/config. Every example has different database.

Reply:
ohno, we need to normalize it doing some ADR or something like this, don't you agree?